pytorch_toolkit/object_detection/custom-detection/problem.yml

```diff
@@ -0,0 +1,9 @@
+description:  >-
+  Custom object detection is a computer technology being used in a variety of applications
+  that identifies custom object in digital images. You need to define your set of detection classes.
+favorite: false
+imagesUrls:
```

Comment:
@dmitryagapov is it needed for custom detector in the platform? Can I remove this?

Reply:
It is not used. Removed